ui/raidboss/data/05-shb/trial/diamond_weapon-ex.txt

```diff
@@ -0,0 +1,116 @@
+### Diamond Weapon Ex
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# -ii 5FEB 5FBE 5FAA 5FEC 5FBA 5FFD 5FBB 6055 6159 5FF9 5FCD 5FFA 5FF9 612E 5FB3 5FCE 5FD2 5FFE 5FD3 5FB4 5FB6 5FB1 5FB0 5FBF 5FAE 5FA6 5FC3 5FC9 5FCA 5FC4
+
+### Phase 1
+# There are no autoattacks from the boss until after the initial raidwide, so nothing to sync against for autos
+0 "Start" sync /Engage!/ window 1,1
+21.3 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/ window 25,10
+
+# Adamant Purge is a mini-phase that will be either stack, spread, or "nothing"
+# 5FA2 = Spread (Homing Laser, 5FA0)
+# 5F9B = "nothing" (Diamond Rain, 5FA7)
+# 5FA5 = Stack (Diamond Flash, 5FA1)
+# This mechanic repeats in phase 3 with additional bits and an extra tracking If circle aoe
+# @TODO: Someone better at timelines than me, sync these to mini-phases instead
+36.5 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/ window 20,5
+46.7 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+49.9 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
+67.1 "Photon Burst" sync /:The Diamond Weapon:5FA8:/
+83.4 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/
+93.6 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+96.8 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
+104.8 "Diamond Rain" sync /:The Diamond Weapon:5FA7:/
+120.0 "Adamant Purge" sync /:The Diamond Weapon:5F(9[ABCD]|A[2345]):/
+130.2 "Claw Swipe" sync /:The Diamond Weapon:5F9[EF]:/
+133.7 "Spread/Stack?" sync /:The Diamond Weapon:5FA[01]:/
+151.4 "Photon Burst" sync /:The Diamond Weapon:5FA8:/
+
+# Phase transition
+167.7 "Code Chi-Xi-Stigma" sync /:The Diamond Weapon:5FAD:/
+183.3 "Auri Cyclone" sync /:The Diamond Weapon:5FB9:/
+184.3 "Auri Cyclone" sync /:The Diamond Weapon:5FD1:/
+188.0 "Diamond Shot" sync /:Articulated Bit:5FC0:/
+
+### Phase 2
+210.0 "Outrage" sync /:The Diamond Weapon:5FBC:/
+218.2 "Outrage" sync /:The Diamond Weapon:5FBC:/
+229.6 "Auri Arts" sync /:The Diamond Weapon:5FAF:/
+236.4 "Auri Arts" sync /:The Diamond Weapon:5FB2:/
+250.4 "Outrage" sync /:The Diamond Weapon:5FBC:/
+257.6 "Auri Doomstead" sync /:The Diamond Weapon:5FBD:/
+271.8 "Auri Arts" sync /:The Diamond Weapon:5FB5:/
+272.8 "Auri Arts" sync /:The Diamond Weapon:5FCF:/
+277.0 "Vertical Cleave" sync /:The Diamond Weapon:5FB7:/
+278.7 "Vertical Cleave" sync /:The Diamond Weapon:5FD0:/
```

Comment:
I think it's a little confusing to have two ability names in a row that do different things.  I would suggest either (1) dropping one of them (5FB7 is the cast, 5FD0 is the knockback) or (2) renaming 5FD0 to `--knockback--`.  I know I suggested using ability names above, but I think a double ability name is one place to ignore that rule.

Similarly, the 5FB5/5FCF Auri Arts above, which are both part of the same mechanic from the player's perspective.  (Does that always happen, since there's an optional extra jump sometimes on that one?)